.gitignore

```diff
@@ -1,144 +0,0 @@
-# Build Folders (you can keep bin if you'd like, to store dlls and pdbs)
```

Comment:
This needs to be reverted.

Reply:
there, I think I fixed it. stil new to Visual Studio and GitHub